app/views/form/guidance/_financial_calculations_shared_ownership.html.erb

```diff
@@ -1,10 +1,22 @@
+<% mortgage_page = log.form.get_question("mortgage", log).page %>
+<% deposit_page = log.form.get_question("deposit", log).page %>
+<% equity_page = log.form.get_question("equity", log).page %>
+<% value_page = log.form.get_question("value", log).page %>
 <%= govuk_details(summary_text: "How the financial values are calculated") do %>
   <p class="govuk-body">
-  The mortgage amount (<%= govuk_link_to "Q92", send("#{log.class.name.underscore}_#{log.form.get_question('mortgage', log).page.id}_path", log) %>),
-   cash deposit (<%= govuk_link_to "Q95", send("#{log.class.name.underscore}_#{log.form.get_question('deposit', log).page.id}_path", log) %>)
-   and cash discount (<%= govuk_link_to "Q97", send("#{log.class.name.underscore}_#{log.form.get_question('cashdis', log).page.id}_path", log) %>)
-   added together must equal
-   the purchase price (<%= govuk_link_to "Q88", send("#{log.class.name.underscore}_#{log.form.get_question('value', log).page.id}_path", log) %>)
-   multiplied by the percentage equity stake (<%= govuk_link_to "Q89", send("#{log.class.name.underscore}_#{log.form.get_question('equity', log).page.id}_path", log) %>)
+    <% if log.type == 18 %>
```

Comment:
This condition shows one guidance text for Social HomeBuy (log type: 18) with the cash discount and another guidance text for all other shared ownership types without the cash discount.

Reply:
Conditional now just used for the first part of the guidance text